internal/pkg/cli/interfaces.go

```diff
@@ -243,6 +244,10 @@ type wsPipelineWriter interface {
 	WritePipelineManifest(marshaler encoding.BinaryMarshaler, name string) (string, error)
 }
 
+type wsPipelineLister interface {
```

Comment:
How about using the existing interface's (`wsPipelineGetter`) `ListPipelines()` method instead of creating new interface? Or there is specific use of this separate interface in the future?

Reply:
Resolved in https://github.com/aws/copilot-cli/pull/3401#discussion_r836934744